lib/diagnostic.js

```diff
@@ -51,6 +97,10 @@ db.dropDatabase();
 // House Stark.
 
 // your code begins here
+db.houses.deleteMany(
```

Comment:
This will not delete anything since you made a typo ("Start" instead of "Stark") when you created the collection.

```javascript
db.houses.deleteOne({
  name: 'House Stark'
});
```